.github/workflows/release.yml

```diff
@@ -62,4 +62,4 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
           short-description: ${{ github.event.repository.description }}
-          repository: michalmrazek9/wanna-ml
\ No newline at end of file
+          repository: michalmrazek9/wanna-ml
```

Comment:
Can we somehow easily change this? Or we don't care too much about the docker image?

Reply:
Gen does not have a dockerhub account as far as I know.

I will delete this